src/Workspaces/Core/Portable/LanguageServices/SyntaxFactsService/ISyntaxFactsServiceExtensions.cs

```diff
@@ -111,5 +111,10 @@ public static SyntaxToken GetOperatorTokenOfBinaryExpression(this ISyntaxFactsSe
             syntaxFacts.GetPartsOfBinaryExpression(node, out _, out var token, out _);
             return token;
         }
+
+        public static bool IsAnonymousOrLocalFunctionStatement(this ISyntaxFactsService syntaxFacts, SyntaxNode node)
```

Comment:
This name sounds a little odd. I would read this as: is "anonymous (function statement) or local function statement". What about just IsAnonymousOrLocalFunction?

Reply:
That name sounds good.  Having Expression/Statement in the name def makes it read a bit odd.  So i nthis case, it seems ok to drop it.

Reply:
Yes but it sounds like we're asking whether something is an anonymous function statement, which doesn't make sense. This should be either `IsAnonymousOrLocalFunction` or `IsAnonymousFunctionExpressionOrLocalFunctionStatement` (obviously I would prefer the first one).

Reply:
Sorry, i wasn't being clear.  When i said "that name sounds good", i meant i liked "IsAnonymousOrLocalFunciton".  That name sounds good and we should use that instead :)

Reply:
Sorry, I misread your comment...

Reply:
I often respond as if we were having a face-to-face conversation.  So you said: 

> What about just IsAnonymousOrLocalFunction?

And i was responding 

> That name sounds good. 

Directly to that part :)    I need to be better about making it clearer since my responses can be read against the entirety of your post and there can be a lot of ambiguity there.  :)

Reply:
My first sentence was "This name sounds a little odd" and you responded "That name sounds good." so I assumed it was in response to that.

Reply:
(but then I apparently ignored the rest of your comment because I'm stupid)